library/src/main/java/io/github/boguszpawlowski/composecalendar/Calendar.kt

```diff
@@ -73,6 +74,7 @@ public fun SelectableCalendar(
   today: LocalDate = LocalDate.now(),
   showAdjacentMonths: Boolean = true,
   horizontalSwipeEnabled: Boolean = true,
+  scrollWeekDays: Boolean = true,
```

Comment:
In order to make it more similar to the other boolean flag, I would call it:
`weekDaysScrollEnabled`